### PR TITLE
theme: change the style of modals

### PR DESCRIPTION
**Pull request by Dinika**
## Checklist:
- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [ ] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [ ] I've added any new docs if API/utils methods were added.
- [ ] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Dinika** [comment]
feedback
This is what it'll look like

**Comment by jacquerie** [comment]
This reminds me of https://github.com/inspirehep/inspire-next/pull/888, where I tried to make all modals look the same, and of https://github.com/inspirehep/inspire-next/pull/599, where some screenshots show how the modal used to look when I originally created it : )

In fact, I believe we have other modals that could benefit from similar changes.

**Comment by Dinika** [comment]
Aah! Okay, I'll update this PR to make these changes to the e-mail request modal and others as well.

**Comment by Dinika** [comment]
@jacquerie  I noticed you added the send paper plane icon in the button. I removed that in this PR. Do you suggest I bring it back?

**Comment by jacquerie** [comment]
> Do you suggest I bring it back?

I trust your design sense a lot more than mine!

**Comment by jmartinm** [comment]
@Dinika indeed here best thing to do would be to modify the style of all modals to be the same. So instead of adding the CSS rule for `#feedbackModalHeader`, apply it to the Bootstrap classes.

Once you modify the style, check for example, the modals in `/literature/new` (after importing) and the modal that appears when exporting in https://labs.inspirehep.net/search to make sure that they are all consistent.

**Comment by Dinika** [comment]
@jmartinm Updated this PR as per the suggestions.

File: inspirehep/modules/theme/static/scss/feedback/modal.scss [hunk 1]
```diff
@@ -18,6 +18,53 @@
...
+      color: white; 
+      opacity: 1;
+  }
+}
+
+.modal-dialog {
+  width: 60%;
+  margin: 0px;
+  position: fixed;
+    top: 30%;
```
Comment (jacquerie) [inline inspirehep/modules/theme/static/scss/feedback/modal.scss]: Indentation is off here, but that's ok: the real issue to fix here is adding some CSS linting step to our build. Perhaps we can postpone this to the general rearchitecture of the UI code.

**Pull request merged**